web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -320,4 +321,14 @@ export class FormFieldEditorComponent implements OnInit, OnChanges, OnDestroy {
       editor.optionGroup.markAllAsTouched();
     });
   }
+
+  private focusNewOption(index: number): void {
+    // setTimeout is to added to wait for the option to load.
+    setTimeout(() => {
+      if (this.optionEditors) {
+        const option = this.optionEditors.find((_val, ind) => ind === index);
```

Comment:
Rather than passing an index and calling "find" can we just use `this.optionEditors.length + 1`?